d2renderers/d2svg/d2svg.go

```diff
@@ -680,17 +735,19 @@ func render3dRect(targetShape d2target.Shape) string {
 			fmt.Sprintf("%d,%d", v.X+targetShape.Pos.X, v.Y+targetShape.Pos.Y),
 		)
 	}
+	// TODO make darker color part of the theme? or just keep this bypass
 	darkerColor, err := color.Darken(targetShape.Fill)
 	if err != nil {
 		darkerColor = targetShape.Fill
 	}
-	sideShape := targetShape
+	sideShape := svgstyle.NewThemableElement("polygon")
 	sideShape.Fill = darkerColor
-	sideShape.Stroke = "none"
-	renderedSides := fmt.Sprintf(`<polygon points="%s" style="%s" mask="url(#%s)"/>`,
-		strings.Join(sidePoints, " "), sideShape.CSSStyle(), maskID)
+	sideShape.Points = strings.Join(sidePoints, " ")
+	sideShape.Mask = fmt.Sprintf("url(#%s)", maskID)
```

Comment:
i think `sideShape.MaskID`, and then you apply this in render

Reply:
Alright, but maybe make it a method (`SetMaskUrl(url string)`) so every property of ThemableElement still represents an element property.